apis/stats_api.py

```diff
@@ -76,14 +81,30 @@ def get_daily_stats():
         SUM(daily_plays) OVER (ORDER BY day) AS total 
     FROM data
     '''
-        
+    
+    queries['daily_started_sprints'] = '''
```

Comment:
I'm not really sure this gives us much more meaningful signal than what we already have in `daily_sprints` lines 68-83. Additionally, this won't do anything past schema v2.1 since we record `start_time` and `end_time` even if the user has not made it past the countdown screen. 

<img width="1463" alt="Screen Shot 2022-08-02 at 4 06 35 PM" src="https://user-images.githubusercontent.com/17424008/182489310-24a5d525-1513-4b88-8337-e234313294cf.png">

top-to-bottom:
1. countdown button pressed @ 4s
2. countdown button pressed @ .9s
3. user exited during countdown @ .2s  (notice loadTime ≈ timeReached)
4. user exited during countdown @ 1s
5. user exited during coundown @ 4s

If we want to capture the amount of users that quit during countdown (which I think is the actual underlying stat this would uncover), I would take one of these approaches:

a) Change play_time to be an explicit 0 for users that did not make it past the countdown screen, and then query for that (+ `path` length of 1). I think @dqian3 and @mliu59 made the decision to still capture the countdown time as play_time, however.

b) Check for a `path` length of 1 and a difference between `loadTime` and `timeReached` < some epsilon.

c) Introduce a new field into our schema flagging if the user made it past countdown. 

Of these, option (c) feels the most robust, at the expense of adding another column. (extra ~265 kB as of writing this, so negligible).

Reply:
Good point, I definitely got misleaded by some of the data in my database, so that is on me. I reverted the changes for now, and I will probably revisit this once we come up with a solution.